src/components/Content.js

```diff
@@ -1,37 +1,67 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 const Content = ({issues}) => {
 
+    const [isShown, setIsShown] = useState(false);
+    const [author, setAuthor] = useState([])
+
     let count=0,count2 = 0, active, closed;
     active = issues.filter(item => item.state === 'open');
     closed = issues.filter(item => item.state === 'closed');
     count = active.length;
     count2 = closed.length;
+    
+    const loadAuthor = (term) => {
+        console.log(term)
+    }
+
+    useEffect(() => {
+
+    }, [])
+
+    const renderedList = issues.map((issue) => {
+        let labelName = '';
+        
+        issue.labels.map((label) => {
+            labelName = label.name
+         })
 
-    const renderedList = issues.map((issue, index) => {
         return(
+          
             <tr key={issue.id}>
                 <td className="collapsing">
                     <i className="dot circle outline large green icon"> </i>
-                    {issue.title}
+                    <a href={issue.html_url} style={{ color:'white' }} target="_blank" alt="issues"> {issue.title}</a>
```

Comment:
You should create your detail page. Dont redirect it to github detail

Reply:
ok sir :)